### DISCUSSION TITLE
The static or unmount props on ComboboxOptions not working.

### DISCUSSION THREAD

**Discussion by prashank25** [tailwindlabs/headlessui#1324] [Help] [answered]
**What package within Headless UI are you using?**

@headlessui/vue

**What version of that package are you using?**

v1.5.0

**What browser are you using?**

Chrome v100

**Reproduction URL**

https://codesandbox.io/s/priceless-napier-5u7vuy?file=/src/components/Combobox.vue

**Describe your issue**

The static or unmount props on ComboboxOptions does not seem to do anything. See the codesandbox with both props, looks like a bug but maybe I am missing something?

**Comment by prashank25** [comment, marked as answer]
I dug some more and my specific issue was because I was using the `TransitionRoot` component from headlessui. However, the `TransitionRoot` works with `ListboxOptions` but not `ComboboxOptions`, codesandbox is updated to reflect that. I don't know if that's the expected behavior or not.

 In summary the `static` prop works with the vue default `transition`, which solves my issue.

**Reply by arukompas** [reply]
this worked great, thank you!

Remember to change the class props on the Vue.js `Transition` element, they're slightly different from those of Headless UI.

https://vuejs.org/guide/built-ins/transition.html#css-based-transitions
